itests/qtest/src/test/java/org/apache/hadoop/hive/cli/TestMiniDruidCliDriver.java

```diff
@@ -29,7 +30,7 @@
 
 import java.io.File;
 import java.util.List;
-
+@Ignore("Flaky due to HIVE-24816")
```

Comment:
Do you know why the test is flaky because of Jackson version change?

Reply:
Yes, this issue is not seen in 2.10.5 jackson version, but it is being seen in every available version post 2.10.5. The error that I'm seeing is: Cannot create PoolableConnectionFactory (java.net.ConnectException : Error connecting to server localhost on port 8,084 with message Connection refused (Connection refused) This error is originated from one of the jackson components in DruidCliDriver. So I had to disable these tests